vector/src/main/java/im/vector/app/features/invite/VectorInviteView.kt

```diff
@@ -56,7 +56,10 @@ class VectorInviteView @JvmOverloads constructor(context: Context, attrs: Attrib
         views.inviteRejectView.commonClicked = { callback?.onRejectInvite() }
     }
 
-    fun render(sender: RoomMemberSummary, mode: Mode = Mode.LARGE, changeMembershipState: ChangeMembershipState) {
+    fun render(sender: RoomMemberSummary, mode: Mode = Mode.LARGE, changeMembershipState: ChangeMembershipState, alreadyAccepted: Boolean) {
+        if (alreadyAccepted) {
+            callback?.onAcceptInvite()
+        }
```

Comment:
I would not change anything in this class, but rather do the logic to `TimelineViewModel` and `RoomDetailViewState` which has a constructor taking `TimelineArgs` in parameter.